migrations/versions/03ceab80c865_initial_revision.py

```diff
@@ -17,7 +17,6 @@
 # from sqlalchemy import engine_from_config
 # from sqlalchemy import inspect
 
-from amanuensis.models import migrate
```

Comment:
I think this is need for retrocompatibility, so you should probably put this back.

Reply:
I added it back